tests/test_quantity.py

```diff
@@ -520,6 +520,94 @@ def test_power_56():
     assert qtm.unit == ""
 
 
+def test_ge_57():
+    x = q.Quantity(10.5, "cm")
+    y = q.Quantity(10.5, "m")
+    z = q.Quantity(10.5, "g")
+    r = q.Quantity(10.5, "")
+
+    with pytest.raises(ValueError) as e_info:
+        assert x >= z
+        assert x >= y
+        assert 5.0 >= r
+
+    with pytest.raises(TypeError) as e_info:
+        assert x >= 5.0
+
+
+def test_gt_59():
```

Comment:
Did you want to test that the operator actually works?  Here you are testing the exceptions so far, which is good...

Reply:
We have already confirmed the operator's working, so only testing the exceptions here. Thank you.

Reply:
OK, staring at the unit tests I cannot see which unit test confirms they are working.  Am I blind?

Reply:
I mean Krishnan and I, checked the both left and right equivalents locally, so I have not explicitly added them in these unit tests.

But now I will add them too.

Thank you.

Reply:
Done. Thank you.